Dockerfile

```diff
@@ -47,8 +47,8 @@ WORKDIR /anvill
 COPY . ./
 
 ENV PATH="${LIBRARIES}/llvm/bin:${LIBRARIES}/cmake/bin:${LIBRARIES}/protobuf/bin:${PATH}"
-ENV CC="${LIBRARIES}/llvm/bin/clang"
-ENV CXX="${LIBRARIES}/llvm/bin/clang++"
```

Comment:
Why aren't we using the version of `clang` from `cxx-common`?

Reply:
cxx-common is built with `/usr/bin/clang`, so I think it's best to have everything consistent.

Reply:
Okay, that should be doable. I'll try to get this working in Docker

Reply:
I pushed a commit to install `clang` during the `deps` stage of Docker, but it seems there are some other errors now regarding dependencies.